test/jdk/ProblemList.txt

```diff
@@ -497,6 +497,7 @@ java/lang/invoke/LFCaching/LFMultiThreadCachingTest.java        8151492 generic-
 java/lang/invoke/LFCaching/LFGarbageCollectedTest.java          8078602 generic-all
 java/lang/invoke/lambda/LambdaFileEncodingSerialization.java    8249079 linux-x64
 java/lang/invoke/RicochetTest.java                              8251969 generic-all
+jni/nullCaller/NullCallerTest.java                              8287745 generic-all
```

Comment:
Currently only fails on linux-x64, but that's in Tier1 so I don't know if it will
fail on other configs in higher tiers.

Reply:
I tried to place safe to exclude it on all platforms.